### PR TITLE
Rename ReportGenerationQuery to ReportQuery

### PR DESCRIPTION
**Pull request by chancez**
Once ReportPrometheusQuery is gone, we can rename ReportGenerationQuery to ReportQuery to be more concise, and shorter.

### CODE REVIEW

File: Documentation/reportdatasources.md [hunk 1]
```diff
@@ -145,7 +145,12 @@ spec:
 [default-storage-location]: storagelocations.md#default-storagelocation
 [architecture]: metering-architecture.md
 [presto-types]: https://prestodb.io/docs/current/language/types.html
+<<<<<<< HEAD
 [reportgenerationqueries]: reportgenerationqueries.md
 [reportgenerationquery]: reportgenerationqueries.md
 [query-inputs]: reportgenerationqueries.md#query-inputs
+=======
```
Comment (timflannagan1) [inline Documentation/reportdatasources.md]: The rebase seems to have included this somehow

Comment (timflannagan1) [inline Documentation/reportdatasources.md]: The rebase somehow included this

Comment (timflannagan1) [reply Documentation/reportdatasources.md]: That was a poor line # to place that comment on, here's the full block:
```
<<<<<<< HEAD
[reportgenerationqueries]: reportgenerationqueries.md
[reportgenerationquery]: reportgenerationqueries.md
[query-inputs]: reportgenerationqueries.md#query-inputs
=======
[reportqueries]: reportqueries.md
[query-inputs]: reportqueries.md#query-inputs
>>>>>>> *: Rename ReportGenerationQuery to ReportQuery
[prestotables]: prestotables.md
```

File: Documentation/reportqueries.md [hunk 2]
```diff
@@ -34,18 +34,18 @@ Most of these functions are for referring to other resources such as `ReportData
 
 #### Query Inputs
 
-Within the template context, the `.Report.Inputs` template variable contains a map where each key is the name of an input defined by the ReportGenerationQueries `spec.inputs`.
+Within the template context, the `.Report.Inputs` template variable contains a map where each key is the name of an input defined by the ReportQueries `spec.inputs`.
```
Comment (timflannagan1) [inline Documentation/reportqueries.md]: ReportQueries -> ReportQuery's `spec.inputs`?

File: Documentation/reportqueries.md [hunk 3]
```diff
@@ -34,18 +34,18 @@ Most of these functions are for referring to other resources such as `ReportData
...
 The value of an input depends on a if the user specified a value in their `Report` or `ReportDataSource`, or if there is a default value defined.
 
-- Depending on what is referencing the ReportGenerationQuery: The value comes from a `Report`'s `spec.inputs` or from a `ReportDataSource`'s `spec.generationQueryView.inputs`.
-- If not provided, it will use the `default` value as defined the the ReportGenerationQueries `spec.inputs`, if one exists.
+- Depending on what is referencing the ReportQuery: The value comes from a `Report`'s `spec.inputs` or from a `ReportDataSource`'s `spec.reportQueryView.inputs`.
+- If not provided, it will use the `default` value as defined the the ReportQueries `spec.inputs`, if one exists.
```
Comment (timflannagan1) [inline Documentation/reportqueries.md]: Same thing here, but also there's a duplicate "the", `as defined the the ReportQueries`

File: Documentation/reportqueries.md [hunk 4]
```diff
@@ -54,7 +54,7 @@ For each of these types, the behavior varies:
 - `time`: A string value is parsed as an RFC3339 timestamp. Within the template context, the variable with be a Go [time.Time][go-time] object.
 - `int`: An int value is passed through as a Go [int][https://golang.org/pkg/builtin/#int].
 - `ReportDataSource`: A string value referencing the name of a [ReportDataSource][reportdatasource] within the same namespace as the query. When this query is referenced by a Report or ReportDataSource, all `ReportDataSource` inputs are validated by checking that all the ReportDataSources specified exist.
-- `ReportGenerationQuery`: A string value referencing the name of a [ReportGenerationQuery][reportgenerationquery] within the same namespace as the query. When this query is referenced by a Report or ReportDataSource, all `ReportGenerationQuery` inputs are validated by checking that all the ReportGenerationQuerys specified exist.
+- `ReportQuery`: A string value referencing the name of a [ReportQuery][reportquery] within the same namespace as the query. When this query is referenced by a Report or ReportDataSource, all `ReportQuery` inputs are validated by checking that all the ReportQuerys specified exist.
```
Comment (timflannagan1) [inline Documentation/reportqueries.md]: This seems off too `checking that all the ReportQuerys specified exist.`

File: Documentation/reportdatasources.md (hunk 1 above)
Comment (chancez) [reply Documentation/reportdatasources.md]: Oh I probably messed that up in a different rebase.

**Pull request merged**